### DISCUSSION TITLE
posts/phobia-workshop/

### DISCUSSION THREAD

**Discussion by giscus** [BYJRK/byjrk.github.io#9] [Announcements]
# posts/phobia-workshop/

这次我们来尝试使用响应式编程来模拟一个有趣的情景短剧《恐惧症研讨会》，并借助各种运算符来模拟剧中每个人的行为逻辑

https://blog.coldwind.top/posts/phobia-workshop/

**Comment by yebei199** [comment]
没有在b站找到对应的视频, 只找到一个py版本的

**Reply by BYJRK** [reply]
就是没有对应视频，还没做😂

**Comment by yisen** [comment]
Jim那里是应该发出aagh吧？

**Reply by BYJRK** [reply]
是的，文章里错写成sorry了😂
